src/ft2font.cpp

```diff
@@ -258,8 +258,11 @@ FT2Font::get_path(std::vector<double> &vertices, std::vector<unsigned char> &cod
 FT2Font::FT2Font(FT_Open_Args &open_args,
                  long hinting_factor_,
                  std::vector<FT2Font *> &fallback_list,
-                 FT2Font::WarnFunc warn)
-    : ft_glyph_warn(warn), image(), face(nullptr)
+                 FT2Font::WarnFunc warn, bool warn_if_used)
+    : ft_glyph_warn(warn), warn_if_used(warn_if_used), image(), face(nullptr),
+      hinting_factor(hinting_factor_),
```

Comment:
```suggestion
      // set a default fontsize 12 pt at 72dpi
      hinting_factor(hinting_factor_),
```

and presumably the comment should be removed below?

Reply:
No, that had nothing to do with the hinting factor, which is why I left it there.